src/OpenApi/JsonSchema/Normalizer/SchemaNormalizer.php

```diff
@@ -38,14 +38,20 @@ public function denormalize($data, $class, $format = null, array $context = [])
         if (!is_object($data)) {
             return null;
         }
+
+        $object = new \Jane\OpenApi\JsonSchema\Model\Schema();
+        $data = clone $data;
```

Comment:
Same here, you can't edit this file manually

Reply:
I saw the "do not edit" hint but still wanted to solve this and raise awareness